### DISCUSSION TITLE
CentOS 6.5安装成功后配置页面缺失DNS服务商选项等一些重要信息，不能使用，急、急、急！

### DISCUSSION THREAD

**Discussion by franklinqu** [jeessy2/ddns-go#1028] [Q&A]
CentOS 6.5安装ddns-go_6.1.2_linux_x86_64.tar.gz，可以加载，但Web配置页面缺失DNS服务商选项等一些重要信息，不能使用，急、急、急！具体见下图：
![sunlogin_20240225013812_0](https://github.com/jeessy2/ddns-go/assets/145885829/2788cdf1-28a1-46bf-b8ed-4ec65768e755)

**Comment by jeessy2** [comment]
换个浏览器

**Comment by franklinqu** [comment]
使用了IE和360安全浏览器，是一样的。

**Reply by jeessy2** [reply]
换个最新的

**Comment by franklinqu** [comment]
换了一台机器，用edge浏览器（OS：Windows Server 2022）打开是正常的，先前的IE和360安全浏览器的操作系统是Win7，会不会与OS有关系？

**Comment by jeessy2** [comment]
严格来说没有
